Journalator/Source/SlashCmd.lua

```
@@ -57,11 +57,86 @@ function Journalator.SlashCmd.Debug(...)
+      Journalator.Utilities.Message("Invalid investment amount")
+      return
+    end
+    
+    local portfolioId = Journalator.InvestONator.CreatePortfolio(name, investmentAmount)
+    Journalator.Utilities.Message("Created portfolio '" .. name .. "' with " .. Journalator.InvestONator.FormatGold(investmentAmount) .. " investment")
+    
+  elseif command == "add" then
+    local portfolioId = tonumber(select(2, ...))
+    local itemName = select(3, ...)
+    local amount = select(4, ...)
+    
```

Luacheck (1.2.0)

[warning] line contains only whitespace

(W611)